controllers/tab/issueEntry/form/IssueEntryPublicationMetadataForm.inc.php

```diff
@@ -300,7 +303,6 @@ function execute($request) {
 					$publishedArticle = $publishedArticleDao->newDataObject();
 					$publishedArticle->setId($submission->getId());
 					$publishedArticle->setIssueId($issueId);
-					$publishedArticle->setDatePublished(Core::getCurrentDate());
```

Comment:
What about article that were not scheduled to an issue but are scheduled right now -- their date_published would be empty which is not good I believe -- the current date i.e. the date when an issue is assigned should be there per default, I believe. I can double check, how is it in the current master branch, but I am pretty sure it was like this...